support/ci_logger/src/lib.rs

```diff
@@ -83,3 +92,9 @@ pub fn init(log_env_var: &str) -> Result<(), log::SetLoggerError> {
     log::set_boxed_logger(Box::new(AdoLogger::new(log_env_var)))
```

Comment:
Given that `AdoLogger::new` is only called here, can we just lift the env query into this function and update `new` to take the value as an `Option<&str>`?